app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class NotesController < ApplicationController
+  before_action :find_pregnancy only: [:create]
```

Comment:
I think this might be missing a comma between the `find_pregnancy` and `only`? Small quibble though!